extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -144,9 +144,39 @@ tags: { # amp-list-load-more
     name: "load-more-end"
     value: ""
     mandatory_oneof: "['load-more-button', 'load-more-failed', 'load-more-end', 'load-more-loading']"
-  }
+  },
 }
 
+# The button element variant allowed in amp-list-load-more.
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
```

Comment:
Are you certain about this list. It seems the various formats are rather opinionated regarding amp-list. @fstanis for email in particular, but ADS probably also doesn't make sense. I don't know about ACTIONS.

Maybe just start with AMP only and see where that gets us?

Reply:
good point. Just AMP it is.